tekton/images/test-runner/setup-kind.sh

```diff
@@ -83,36 +83,25 @@ if [[ "${E2E_ENV}" != "" && ! -f "${E2E_ENV}" ]]; then
 fi
 
 # The version map correlated with this version of KinD
-KIND_VERSION="v0.11.1"
```

Comment:
do we still need this env var?

Reply:
I don't think it's used anywhere. Instead, the [arg with same name](https://github.com/tektoncd/plumbing/pull/1507/files#diff-e5330bef7d087643ebe22142213af1210c100d8ce4029639aa371a6c3b81d04bR169) defined in dockerfile is actually used.